src/ResourceManager/Compute/Commands.Compute/AzureRM.Compute.psd1

```diff
@@ -181,7 +181,8 @@ CmdletsToExport = 'Remove-AzureRmAvailabilitySet', 'Get-AzureRmAvailabilitySet',
                'New-AzureRmGalleryImageDefinition', 'Update-AzureRmGalleryImageDefinition',
                'Get-AzureRmGalleryImageDefinition', 'Remove-AzureRmGalleryImageDefinition',
                'New-AzureRmGalleryImageVersion', 'Update-AzureRmGalleryImageVersion',
-               'Get-AzureRmGalleryImageVersion', 'Remove-AzureRmGalleryImageVersion'
+               'Get-AzureRmGalleryImageVersion', 'Remove-AzureRmGalleryImageVersion',
+               'Add-AzureRmVmssVMDataDisk', 'Remove-AzureRmVmssVMDataDisk'
```

Comment:
@hyonholee please make sure these cmdlets are added to `Az.Compute.psd1` as well